lib/spack/spack/main.py

```diff
@@ -409,7 +417,7 @@ def make_argument_parser(**kwargs):
         help="lines of profile output or 'all' (default: 20)")
     parser.add_argument(
         '-v', '--verbose', action='store_true',
-        help="print additional output during builds")
+        help="print additional, verbose output")
```

Comment:
```suggestion
        help="tee build system output to stdout")
```

Reply:
not sure everyone knows `tee` -- `echo`?  `print`?

Reply:
I removed the reference to `builds` under the assumption verbose *could* apply to any Spack operation. 

 I'll restore the original message.

Reply:
Thanks! `-v` on `install` is really just the build output

Reply:
Restored.